tasks/selinux.yml

```diff
@@ -1,11 +1,14 @@
 # SPDX-License-Identifier: MIT
 ---
+- name: Populate service facts
+  service_facts:
+
 - block:
     - name: Set the fence-virt/fence-agents port to _ha_cluster_selinux
       set_fact:
         _ha_cluster_selinux:
           - {'ports': '1229', 'proto': 'tcp', 'setype': 'cluster_port_t',
-             'state': '{{ _state_value }}', 'local': 'true'}
+             'state': 'present', 'local': 'true'}
```

Comment:
I think this should be `'local': true` - that is, the boolean value `true` instead of the string value `'true'` - also below